README.md

```diff
@@ -55,11 +34,14 @@ Example
 `crontab -e`
 ```
 SENTRY_DSN=https://<your_key>:<your_secret>@app.getsentry.com/<your_project_id>
-@reboot cron-sentry ./my-process
+cron-sentry my-process --arg arg2
```

Comment:
Aha, yes! The @reboot line confused me (I'd not seen it before), so I got rid of it (forgetting what is was I expected in the first place). Updated.